### DISCUSSION TITLE
Further planned update policy

### DISCUSSION THREAD

**Discussion by eoverkamp-ionos** [woelper/oculante#403] [General]
Hello there. I was wondering where there is a planned update policy for this project to guarantee support in the forseable future. :)

**Comment by woelper** [comment]
Hi!

What would be part of such an update policy to make it valuable for you and other users?

**Comment by eoverkamp-ionos** [comment]
For me the value of such an update policy would be a statement about the update cycles of this software. Either something like "Updates are not set in a set schedeule." or "We plan to keep this project going as long as XY". Just so one knows wether this project is still being worked on in the future.  Thank yo for your quick reply and please excuse my very belated response.

**Reply by woelper** [reply]
Sure, I think that makes sense!

I'll think of something to put in the readme.

I think if you take a look at the releases tab of this repository you might get an overview of the work and release cadence. I tend to release versions often, but there needs to be some sort of value for the users. I am not a big fan of doing regular updates which don't contain real value.

**Reply by Stoppedpuma** [reply]
@e0vk I put something in the README recently, does this help address your concerns? https://github.com/woelper/oculante?tab=readme-ov-file#updates

**Reply by eoverkamp-ionos** [reply]
Yes this helps! :) Thank you very much for adding this information.
